### ISSUE TITLE
Sort list of models alphabetically, or open model via "open file"-dialog

### ISSUE THREAD

**Issue by Feathergunner** [lumen-org/lumen#79]
To find a specific model if the model directory contains a large number of models can be complicated.

**Comment by asver12** [comment]
Do you mean when searching a model using the input box right next to "Load Model:" ?
It has an "autocomplete" feature so you will see a drop-down of all models with your specific string while you input data. It still could be sorted.

**Comment by Feathergunner** [comment]
Yes, that's what i meant. Because sometimes I don't exactly remember how the name of a model started; in these cases i have to look at the list of all models to remember my naming scheme. And once the list is open, it would be easier and faster if the user could directly select the correct model from a sorted list.
And I think having these kinds of lists sorted makes a better impression, otherwise the software feels unfinished.
